index.md

```diff
@@ -1,4 +1,44 @@
 ---
 layout: docs
-title:  "Using our Collections API"
----
\ No newline at end of file
+title:  "Getting Started with our Collections API"
+---
+
+## What is an API? 
+
+An API (Application Programming Interface) is a set of instructions that tells one software application how to "talk to", or transmit data, to another. If you have used [DIMES](https://dimes.rockarch.org/), our site to enable search and access to our collections, you have seen our API in action. For example, when you initiate a search for something in DIMES, your browser makes a request to the API, which delivers the data as the search results. 
```

Comment:
"used an API" rather than "seen our API in action"? I get what you're going for but the reality is that the API piece of what's happening is pretty invisible...